bin/reth/src/node/mod.rs

```diff
@@ -122,7 +130,12 @@ impl Command {
 
         // Run pipeline
         info!(target: "reth::cli", "Starting sync pipeline");
-        pipeline.run(db.clone()).await?;
+        tokio::select! {
+            res = pipeline.run(db.clone()) => res?,
+            _ = tokio::signal::ctrl_c() => {},
+        };
```

Comment:
for now this looks fine since the pipline is run on the main task.

but ideally, this should be integrated via #1036